packages/preferences/src/browser/views/preference-searchbar-widget.tsx

```diff
@@ -53,19 +55,49 @@ export class PreferencesSearchbarWidget extends ReactWidget {
         }
     }
 
+    protected clearSearchResults = (e: React.MouseEvent): void => {
+        const search = document.getElementById('preference-searchbar') as HTMLInputElement;
+        if (search) {
+            search.value = '';
+            this.search(search.value);
+            this.update();
+        }
+    };
+
+    protected renderOptionContainer(): React.ReactNode {
+        const clearAllOption = this.renderOptionElement();
+        const resultsCount = this.renderResultsCount();
+        return <div className="option-buttons"> {this.searchbarRef.current?.value !== '' ? resultsCount : ''} {clearAllOption} </div>;
+    }
+
+    protected renderResultsCount(): React.ReactNode {
+        return <span
+            className="results-found"
+            title={`${this.searchCount} Results Found`}> {this.searchCount} Results Found </span>;
```

Comment:
For consistency, we should align with vscode:

```suggestion
            title={`${this.searchCount} Settings Found`}> {this.searchCount} Settings Found </span>;
```

<br />

When there are no settings found (0 results), we can display `No Settings Found`.